src/commands/config/index.ts

```diff
@@ -54,6 +57,36 @@ class ConfigCommand extends Command {
               )
           )
       )
+      .addSubcommandGroup((subcommandGroup) =>
```

Comment:
The more subcommand that get added, the more I think we should have a subcommand class. Its just a LOT more work then I feel like putting in just yet.

Reply:
Maybe do that after getting this command finished?